templates/docker-compose.yml.j2

```diff
@@ -13,3 +13,5 @@ services:
       timeout: {{ healthcheck_timeout }} 
       retries: {{ healthcheck_retries }}  
     env_file: "sonda.env"
+    extra_hosts:
+      - '{{ inventory_hostname }}:{{ ansible_local.wireguard.address }}'
```

Comment:
I'm assuming that it's intentional that `inventory_hostname` lacks the `.wg` suffix.

Reply:
But maybe do add a comment explaining why this is necessary.